extensions/grpc/runtime/src/main/java/io/grpc/override/ContextStorageOverride.java

```diff
@@ -0,0 +1,67 @@
+package io.grpc.override;
+
+import io.grpc.Context;
+import io.smallrye.common.vertx.VertxContext;
+import io.vertx.core.Vertx;
+
+/**
+ * Override gRPC context storage to rely on duplicated context when available.
+ */
+public class ContextStorageOverride extends Context.Storage {
```

Comment:
wow that's nasty (in gRPC) , the `io.grpc.override.ContextStorageOverride` hardcoded class.